docs/getting-started/create-pipeline.md

```diff
@@ -0,0 +1,61 @@
+# Creating a Pipeline
+Using the Screwdriver CLI, you can build pipelines locally on your machine.
+
+## Set up Screwdriver CLI
+
+**The Screwdriver CLI does not exist yet, but here is what the instructions would looks like.**
+
+### Installing the CLI
+
+```bash
+$ go get github.com/screwdriver-cd/client
+$ $GOPATH/bin/client
```

Comment:
Discussed this during stand-up. We intend to keep this as-is, until we have a better alternative to how to convey this information.